content/docs/pipelines/caching.md

```diff
@@ -0,0 +1,110 @@
++++
+title = "Caching"
+description = "Getting started with Kubeflow Pipelines step caching"
+weight = 50
++++
+
+Starting from Kubeflow Pipelines 0.4, Kubeflow Pipelines supports step caching capabilities in both standalone deployment and GCP hosted deployment.
+
+## Before you start
+
+This guide tells you the basic concepts of Kubeflow Pipelines step caching and how to use it. 
+This guide assumes that you already have Kubeflow Pipelines installed or want to use standalone or GCP hosted deployment options in the [Kubeflow Pipelines deployment 
+guide](/docs/pipelines/installation/) to deploy Kubeflow Pipelines.
+
+## What is step caching?
+
+Kubeflow Pipelines caching provides step-level output caching. 
```

Comment:
Suggested revision for this paragraph:

Kubeflow Pipelines 0.4.0  provides step-level output caching for pipelines built with the Kubeflow Pipelines SDK. 
Step caching is enabled by default. 
With caching enabled, the system can skip steps that have been executed in previous runs,  which saves time and money.
The outputs of cached steps are reused when the cache key of a step matches the cache key of a previously executed step. 
The cache key is based on the component (base image, command-line, code) and arguments passed to the component (values or artifacts).
If the component is exactly the same and the arguments are exactly the same as in some previous execution, then the step can be skipped and the outputs of the previous execution are used. 
Pipeline authors control how long executions are cached by [setting the maximum cache staleness](/docs/pipelines/caching/#managing-caching-staleness).

Pipelines created with the TensorFlow Extended SDK use a different mechanism to cache step execution.